python/mxnet/numpy_dispatch_protocol.py

```diff
@@ -119,6 +119,7 @@ def _run_with_array_ufunc_proto(*args, **kwargs):
     'var',
     'vdot',
     'vstack',
+    # 'column_stack',
```

Comment:
Why is this commented out?

Reply:
To pass numpy interoperability test temporarily because there's no test now and it will throw error otherwise. I'll add a test later.

Reply:
Please keep this and add some test cases not necessarily from NumPy in `test_numpy_interoperability.py`.